simulation/halsim_ds_socket/build.gradle

```diff
@@ -0,0 +1,27 @@
+description = "A plugin that listens on a socket so that you can use the real Driver Station software to connect to the simulation"
+
+ext {
+    includeWpiutil = true
+    pluginName = 'halsim_ds_socket'
+}
+
+ext.cppfrccomm = { binary->
+    binary.tasks.withType(CppCompile) {
+            binary.cppCompiler.args "-I" + project(':ni-libraries').file('include').absolutePath
+    }
+    if (binary.targetPlatform.operatingSystem.windows) {
+        binary.linker.args "ws2_32.lib"
```

Comment:
We usually do this with a `#pragma lib` in the source file that needs it. I would prefer to do the same thing here.